src/impl_x86__base_implementation.inl

```diff
@@ -1755,6 +1757,7 @@ CacheInfo GetX86CacheInfo(void) {
   LINE(X86_RDRND, rdrnd, , , )                             \
   LINE(X86_DCA, dca, , , )                                 \
   LINE(X86_SS, ss, , , )                                   \
+  LINE(X86_LZCNT, lzcnt, , , )                             \
```

Comment:
nit should be placed after ADX to be order consistent with the X86FeaturesEnum declaration

Reply:
thanks, updated